tests/integration/test_database.py

```diff
@@ -36,7 +36,7 @@ async def test_database_relation(ops_test: OpsTest, mysql_router_charm_series: s
     applications = await asyncio.gather(
         ops_test.model.deploy(
             "mysql",
-            channel="8.0/edge",
+            channel="8.0/stable",
```

Comment:
My thought process was that we are testing the router here and not the server (which is why I thought it might make more sense to bump to `stable`). The test was written when we didnt have a stable risk

I can revert if we find that there's a reason to use the `edge` server

Reply:
tbf, we should be testing `edge` on the bundle, but since we in practical terms are not, I see two ways:
1. test the bundle
2. add router/stable test on server

I prefer the latter - thoughts?

Reply:
addressed in [1c43b68](https://github.com/canonical/mysql-router-operator/pull/80/commits/1c43b686b020e50796565b123229388f34174294). we are running tests against edge of mysql and mysql-test-app